### ISSUE TITLE
Warn when on blocked IP

### ISSUE THREAD

**Issue by RayBB** [fuddl/wd#126]
I didn't realize I was editing from a free vpn (Proton) and my edits appeared to be working in the wikidata extension but not showing up on wikidata website. So I tried to make an edit on the website and it failed because it's blocked. However, the extension still seemed to think the edits went through even after many refreshes. Maybe there is a quick fix to make it so that it warns when your edit was blocked?

Thanks

**Comment by fuddl** [comment]
You could maybe fire a [notification](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Notifications) when an edit was rejected by the wikidata api.

**Comment by RayBB** [comment]
That would be fine for me personally but I suspect some people would rather not give notification permission to an extension for a small use case. An `alert()` on the page could do the job too without mucking about in the UI.

**Comment by fuddl** [comment]
I don't know. An alert interrupts the workflow. If I do a mass edit with 100 statements where each statment's reference is rejected because the domain of a url is blacklisted, one would need to okay 100 alerts or block them completely. 

The nice thing about notifications is that one can just ignore them if one chooses to.

**Comment by RayBB** [comment]
Good point. Perhaps the alert should only happen once per X seconds or something. 
But I agree it's not an ideal solution. I think it would be nice if it showed in the UI before the user tried to make a change but I don't see an API to check if your IP is blocked.

**Comment by fuddl** [comment]
> Good point. Perhaps the alert should only happen once per X seconds or something.

or i could do it once per editing queue. Along the lines of 

> One of your edits has been rejected by Wikidata. Consider login into Wikidata and try again.
